frontend-react/src/hooks/UseOktaMemberships.ts

```diff
@@ -155,7 +160,7 @@ const calculateNewState = (
                 ),
                 initialized: true,
             };
-        case MembershipActionType.ADMIN_OVERRIDE:
+        case MembershipActionType.UPDATE_MEMBERSHIP:
```

Comment:
What's the reason for this renaming?  I feel like `ADMIN_OVERRIDE` makes more sense since the intention here is to update the override value in LocalStorage even though it updates the global membership state all the time [in the reducer](https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/hooks/UseOktaMemberships.ts#L203).

It shouldn't be part of this changeset, but I'm thinking eventually we should keep `global-membership-state` as the canonical "true" user value whereas `global-organization-override` can be anything that an admin would be impersonating.  Our membership hooks/functions look like they already prioritize the override value so that might not be too huge a change.  But right now, it feels like their usage is kind of muddled 🤔 

--- 

I also wonder if we should really be accessing LocalStorage as frequently as we are since its operations are synchronous or if we can persist state in memory -- but this is a really small point in my mind; it's not like our user states are changing a thousand times per second or anything, in which case we may have a severe case of multiple personality disorder.

Reply:
Right, my follow-up point was about splitting up the logic here so that only the `global-organization-override` value would be updated rather than the `global-membership-state` as well -- the latter would essentially be static on login, and the former would change when we impersonate an Organization.  Our in-memory abstractions can factor in both memberships wherein the override would take priority, but I think it'd help to treat them functionally separately in LocalStorage.

Reply:
I might need to refresh my brain on how we're using localStorage, but I don't think having two versions of membership state really serves a benefit, _yet_. Considering there's no real "PrimeAdmins" organization settings, the admins themselves don't benefit from having their default parsed-from-token settings; it's only when they override that they really have any benefit for an admin.

Reply:
Fair point.  I was thinking it could be useful for logical separation of the "canonical" user versus an impersonated user, but to that point, we could just use `memberType` for the time being.  I guess I don't really understand why we keep both `global-organization-override` AND `global-membership-state` when the `activeMembership` stuff is kept the same in both -- seems redundant 🤔

Reply:
Sounds good -- I can do some initial investigation on where it's used and ticket the cleanup work, but for now I'd say we can punt on this point!